yt_dlp/extractor/mave.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+)
+
+
+class MaveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<channel_id>[a-z]+)\.mave\.digital/ep-(?P<short_id>[0-9]+)'
```

Comment:
IMO we should relax the subdomain character class and capture the whole slug
```suggestion
    _VALID_URL = r'https?://(?P<channel>[\w-]+)\.mave\.digital/(?P<id>ep-\d+)'
```